src/storage/btree.c

```diff
@@ -16450,25 +16517,55 @@ btree_find_min_or_max_key (THREAD_ENTRY * thread_p, BTID * btid, DB_VALUE * key,
       goto exit_on_error;
     }
 
-  if (!BTREE_END_OF_SCAN (BTS))
+  if ((mvcc_snapshot = logtb_get_mvcc_snapshot (thread_p)) == NULL)
     {
-      assert (BTS->slot_id > 0);
-      if (spage_get_record (thread_p, BTS->C_page, BTS->slot_id, &rec, PEEK) != S_SUCCESS)
+      ret = er_errid ();
```

Comment:
logtb_get_mvcc_snapshot seems not to set error code. er_errid () always set NO_ERROR

Reply:
will be handled at **exit_on_error** as like other error case.